middle_end/flambda/flambda_utils.mli

```diff
@@ -69,13 +69,9 @@ val make_closure_declaration
   -> region:bool
   -> body:Flambda.t
   -> params:Parameter.t list
-<<<<<<< HEAD
   -> return_layout:Lambda.layout
   -> free_variables:Lambda.layout Variable.Map.t
-||||||| merged common ancestors
   -> stub:bool
```

Comment:
Looks like both we and they removed this parameter. Drop it?

Reply:
Good catch, thanks. That was a mistake.